tsconfig.json

```diff
@@ -1,17 +1,17 @@
 {
   "compilerOptions": {
     "target": "ES2021",
-    "allowJs": true,
```

Comment:
No need to include JS files by default, since we will have all the source in TypeScript